handlers/configuration.go

```diff
@@ -0,0 +1,51 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/penalty-payment-api-core/finance_config"
+	"github.com/companieshouse/penalty-payment-api-core/models"
+	"github.com/companieshouse/penalty-payment-api/common/utils"
+	"github.com/companieshouse/penalty-payment-api/configctx"
+)
+
+type ConfigResponse struct {
+	Type string      `json:"type"`
+	Data interface{} `json:"data"`
+}
+
+// HandleConfiguration returns an HTTP handler that serves configuration data
+func HandleConfiguration(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var response []finance_config.PenaltyConfig
+
+	penaltyConfig := configctx.FromContext(r.Context())
+
+	for _, p := range penaltyConfig.PayablePenaltyConfigs {
+		penaltyValue := *p.Penalty
+		now := time.Now()
+
+		enabledFrom := penaltyValue.EnabledFrom
+		enabledTo := penaltyValue.EnabledTo
+
+		if (enabledFrom != nil && now.After(*enabledFrom)) &&
```

Comment:
I think the enabledFrom condition should be !now.Before so that it is inclusive?

```suggestion
		if (enabledFrom != nil && !now.Before(*enabledFrom)) &&
```